### PR TITLE
Updated all IPs to IP addresses to make spell checker happy

### PR DESCRIPTION
**Pull request by runwuf**
Updated all IPs to IP addresses to get rid of spell check warnings

### CODE REVIEW

File: docs/platform/howto/static-ip-addresses.rst [hunk 1]
```diff
@@ -117,8 +117,8 @@ Remove static IP addresses
 
 .. note::
 
-    To dissociate an IP from a service, the service must either have enough IPs
-    still available, or the ``static_ips`` configuration setting for the
+    To dissociate an IP from a service, the service must either have enough IP addresses
+    still available, or the ``static_IP addresses`` configuration setting for the
```
Comment (ftisiot) [inline docs/platform/howto/static-ip-addresses.rst:121]: Not sure this is correct :D

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @runwuf, I'm not sure one substitution is correct :D

File: docs/platform/howto/static-ip-addresses.rst (hunk 1 above)
Comment (runwuf) [reply docs/platform/howto/static-ip-addresses.rst]: Thank you for catching this one @ftisiot! 😅🙇‍♂️

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**